discovery-provider/src/queries/search_queries.py

```diff
@@ -401,33 +358,34 @@ def submit_and_add(search_type):
 
             if searchKind in [SearchKind.all, SearchKind.tracks]:
                 submit_and_add("tracks")
```

Comment:
Not that it really matters, but we can lower the max_workers in in the threadpool now, since we only ever perform 4 requests in parallel.

Reply:
Ooh that's why it was set at 8. Yup I'll change it.